apidoc.json

```diff
@@ -2,7 +2,8 @@
     "name": "Express Todo API",
     "version": "1.0.0",
     "description": "Documentation for Simple Todo API with Express.js. App is using JWT, which provides authorization token. If user has logged, token is generated and is necessary to any other actions handle with VerifyToken middleware",
-    "template":{
+    "url": "127.0.0.1:3000",
```

Comment:
we can not keep it in repo, on server it will be different

Reply:
what about this?